colm/tableflux/run

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
```

Comment:
Can you add a binary that takes a tableflux input and outputs a flux output? Something like a `colm/cmd/tableflux` package and it takes a file as an argument and outputs the transformation. It would make it a lot easier to experiment with this prototype to be able to see what the code transforms into.

Reply:
I just found this binary. Can you include instructions on using it in the README? I tried to do `./tableflux <file>` and it didn't work, but `./tableflux < <file>` worked.